### PR TITLE
added buildings back in the event module

### PR DESCRIPTION
**Pull request by jannelie**
As a developer I want to have buildings in my believe, so that I know which buildings to demolish

### CODE REVIEW

File: src/main/goal/nl/tudelft/ti2806/service.mod2g [hunk 1]
```diff
@@ -53,8 +53,9 @@ module main{
 		goal(buildTerrace; buildStore; buildSportsCenter), 
 		bel(
 			my_stakeholder_id(StakeholderID),
-			buildings(X), 
-			member(building(BuildingID,_,StakeholderID,_,_,FunctionID,_, _), X), 
+			buildings(X),
```
Comment (nskartoredjo) [inline src/main/goal/nl/tudelft/ti2806/service.mod2g]: rename X

File: src/main/goal/nl/tudelft/ti2806/tygronEvents.mod2g [hunk 2]
```diff
@@ -22,6 +22,14 @@ module tygronEvents {
 		delete(action_logs(Y)) + 
 		insert(action_logs(X)).
 
+	% description: Whenever buildings are percepted old buildings will be deleted and new buildings will be inserted.
+	if
+		percept(buildings(X)),
```
Comment (nskartoredjo) [inline src/main/goal/nl/tudelft/ti2806/tygronEvents.mod2g]: Rename all single letter variables

**Comment by nskartoredjo** [comment]
Approved

**Comment by ardwennem2** [comment]
Approved

**Comment by HarmenKroon** [comment]
Approved

**Pull request merged**